### PR TITLE
docs: add usage notice for CLI analytics

### PR DESCRIPTION
**Pull request by hansl**
This is going to be linked to users who want to know more about
analytics.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by gkalpak** [comment]
So, this is not supposed to show up in the navigation menu, right?
In any case, I would prefer to have this inside a directory (e.g. cli) or at least include `cli` in the file name.

**Comment by hansl** [comment]
It could show up in the navigation, yes, but the URL should be as simple as possible. I'll discuss with Igor.

**Comment by jenniferfell** [comment]
I recommend renaming `aio/content/usage/index.md` to `aio/content/cli/usage-analytics.md`. (or `usage-metrics.md).  Resulting URL will be: `angular.io/cli/usage-analytics`

Reason: I agree with @gkalpak that the association with the CLI is essential in the filename and URL. 
* As a URL, "angular.io/usage" is misleading. This isn't about using Angular. 
* As a directory structure, I think it's important to keep this with the other CLI-tool-specific information, in `angular/content/cli`.

**Comment by hansl** [comment]
@jenniferfell @gkalpak 

I discussed with Igor and the goal here is to have a generic `/analytics` link that would also explain analytics gathering for other things angular related (e.g. we gather angular.io and other websites analytics). `/usage` was too generic, so we landed on `/analytics` instead.

Hope this clears things up. I'm going to try to clean this PR to make it work and get people's reviews today.

**Comment by hansl** [comment]
This is now under the footer's section `Resources`.

File: aio/content/guide/usage-analytics-sharing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Administrators can configure their instances of Angular CLI to share usage analytics with
+third parties. This sharing is separate from and in addition to usage analytics shared
+with Google. It is disabled by default and cannot be enabled at the project level for
+all users. It must be set at the global configuration level for each user to allow
+managed users to opt in and share usage analytics.  
+
+To configure usage analytics sharing, use the `ng config` command to add both the user ID
+and the third-party ID to the user's global Angular CLI configuration. 
+
+- The third-part ID is the tracking identifier of the third-party Google Analytics
```
Comment (alexeagle) [inline aio/content/guide/usage-analytics-sharing.md]: third-part -> third-party

File: aio/content/marketing/analytics.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Usage Metrics Gathering
+You can help the Angular team to prioritize features and improvements by permitting the Angular team to send command-line command usage statistics to Google. The Angular Team does not collect usage statistics unless you explicitly opt in during the Angular CLI installation or upgrade.
+
+## What is collected?
+Usage statistics contain commands and selected flags for each execution and may include, for example:
```
Comment (alexeagle) [inline aio/content/marketing/analytics.md]: should we mention that this starts in CLI version 8? so users don't go looking for the feature before they have updated?

File: aio/content/guide/usage-analytics-sharing.md (hunk 1 above)
Comment (alexeagle) [inline aio/content/guide/usage-analytics-sharing.md]: third-party

**Review by alexeagle — APPROVED** [review]

File: aio/content/guide/usage-analytics-sharing.md (hunk 1 above)
Comment (hansl) [reply aio/content/guide/usage-analytics-sharing.md]: Done.

File: aio/content/marketing/analytics.md (hunk 2 above)
Comment (hansl) [reply aio/content/marketing/analytics.md]: Done.

File: aio/content/guide/usage-analytics-sharing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Usage Analytics Sharing
```
Comment (IgorMinar) [inline aio/content/guide/usage-analytics-sharing.md]: @jenniferfell should we prefix cli docs with `cli-..... . md`? or could we somehow group them together?

**Review by IgorMinar — APPROVED** [review]

File: aio/content/guide/usage-analytics-sharing.md (hunk 3 above)
Comment (jenniferfell) [reply aio/content/guide/usage-analytics-sharing.md]: We have a directory for CLI command-specific things. aio/content/cli

Comment (hansl) [reply aio/content/guide/usage-analytics-sharing.md]: After discussion with Jennifer, It's going to be under `cli/`, and in the TOC under `CLI commands`.

**Review by jenniferfell — APPROVED** [review]

File: aio/content/marketing/analytics.md (hunk 2 above)
Comment (jenniferfell) [reply aio/content/marketing/analytics.md]: Per Stephen and Igor (and best practices industry wide), we don't include version numbers for features. If you're looking at next, then all doc applies to version 8. If you're looking at stable andd later that will be called v7, then the content matches v7 software.

Comment (jenniferfell) [reply aio/content/marketing/analytics.md]: Update: This file is linked site-wide -- not to a specific docs version -- so mentioning version makes sense.

**Pull request closed without merging**